rotkehlchen/api/rest.py

```diff
@@ -1647,6 +1647,21 @@ def process_history(
         result_dict = _wrap_in_result(result=result, message=msg)
         return api_response(result_dict, status_code=status_code)
 
+    @require_loggedin_user()
+    def get_history_actionable_items(self) -> Response:
+        missing_acquisitions = self.rotkehlchen.accountant.pots[0].cost_basis.missing_acquisitions
+        missing_prices = self.rotkehlchen.accountant.pots[0].cost_basis.missing_prices
+
+        processed_missing_acquisitions = process_result_list(missing_acquisitions)
+        processed_missing_prices = process_result_list(missing_prices)
```

Comment:
No need to call `process_result_list`. You already have the serialize function and no unknowns are in there.
```suggestion
        processed_missing_acquisitions = [x.serialize() for x in missing_acquisitions]
        processed_missing_prices = [x.serialize() for x in missing_prices]
```